### ISSUE TITLE
mAP has hardly decreased

### ISSUE THREAD

**Issue by hoopool** [veralauee/DPatch#3]
I have try to run train.py of the project, and run test.py of the project using the latest patch that saved by train.py (without using the untargetd_DPATCH which download from the url that given at the README.md file). 
after 100 epoch, it got 0.7095 mAP, but after 9757 epoch, it still got 0.7040 mAP.
I just wander it there any parameters that I must modified before run train.py? I only modified max_epoch value in cfgs.exps.darknet19_exp1.py to run more epoch.

than you~

**Comment by quahez** [comment]
same problem here. 
I've tried to modify the threshold values, increase the batch size, adjusted the learning rate, momentum, increase the epoch, etc.. but to no avail. The mAP is still way above the 0.6 mark (usually hovers around 0.7). 
Is there a way to verify if the patch (416x416 and reduced before attaching to image) is indeed attached to the image prior to running YOLO?

Thanks!

**Comment by xubo0321** [comment]
> same problem here.
> I've tried to modify the threshold values, increase the batch size, adjusted the learning rate, momentum, increase the epoch, etc.. but to no avail. The mAP is still way above the 0.6 mark (usually hovers around 0.7).
> Is there a way to verify if the patch (416x416 and reduced before attaching to image) is indeed attached to the image prior to running YOLO?
> 
> Thanks!

Did you succeed in training and testing? can you tell me your email, my email is xu_bo@ntu.edu.sg.

**Comment by xubo0321** [comment]
> I have try to run train.py of the project, and run test.py of the project using the latest patch that saved by train.py (without using the untargetd_DPATCH which download from the url that given at the README.md file).
> after 100 epoch, it got 0.7095 mAP, but after 9757 epoch, it still got 0.7040 mAP.
> I just wander it there any parameters that I must modified before run train.py? I only modified max_epoch value in cfgs.exps.darknet19_exp1.py to run more epoch.
> 
> than you~
Can we discuss the method together, I also have some problems, can you email me or tell me your email, my email is xu_bo@ntu.edu.sg.

**Comment by CNHNLP** [comment]
@hoopool @quahez @xubo0321 I also find the same question, have you finished? can you give me some suggestion? thank you

**Comment by chenxiaoyupetter** [comment]
mAP has hardly decreased 
我测试了我认为可以调整的参数，但是实验效果依旧不是很好，包括使用了您这边给出的完整版的参数，请问您这边可以给出完整的参数配置么？

**Comment by yebin999** [comment]
> mAP has hardly decreased
> 我测试了我认为可以调整的参数，但是实验效果依旧不是很好，包括使用了您这边给出的完整版的参数，请问您这边可以给出完整的参数配置么？

你好，你那边程序跑通了吗，能否加个联系方式请教一下，我这边好像缺少作者自己写的模块文件

**Comment by xleslie** [comment]
I have the same problem，have you solved it？

**Comment by chenxiaoyupetter** [comment]
i will help if i can slove the problem



---Original---

**Comment by xleslie** [comment]
> i will help if i can slove the problem
> […](#)
> ---Original--- From: "xleslie"<notifications@github.com&gt; Date: Fri, Sep 18, 2020 18:20 PM To: "veralauee/DPatch"<DPatch@noreply.github.com&gt;; Cc: "chenxiaoyupetter"<413351385@qq.com&gt;;"Comment"<comment@noreply.github.com&gt;; Subject: Re: [veralauee/DPatch] mAP has hardly decreased (#3) I have the same problem，have you solved it？ — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or unsubscribe.

Thanks!I will also try to solve the problem,my email is  bitxyd@gmail.com

**Comment by zxydi1992** [comment]
I have the same problem here.
